app/models/data_source_adapters/redshift_adapter.rb

```diff
@@ -65,16 +62,15 @@ def fetch_columns(table)
             end
           end
         column_type = sql_type == 'character varying' ? "varchar(#{char_length})" : sql_type
-        Column.new(name, column_type, adapter.quote(default), is_nullable)
+        Column.new(name, column_type, default.nil? ? 'NULL' : default, is_nullable)
```

Comment:
I believe this quoting is unnecessary since the return value is only used on our HTML views. Just using the value of `default` would be enough to handle raw values of tables.

The following is a screenshot of an example table.

<img width="1259" alt="スクリーンショット 2024-04-09 20 14 05" src="https://github.com/cookpad/dmemo/assets/4374702/a2d36214-77e8-4268-b392-4b7eafab6b79">